Test for WidgetFooter.jsx is date-sensitive

The test for `WidgetFooter.jsx` depends on the current date and fails after a certain date:
```text
[ERROR]   ● <WidgetFooter /> › should render a widget footer with unlocked
[ERROR] 
[ERROR]     expect(value).toMatchSnapshot()
[ERROR]     
[ERROR]     Received value does not match stored snapshot 1.
[ERROR]     
[ERROR]     - Snapshot
[ERROR]     + Received
[ERROR]     
[ERROR]     @@ -15,11 +15,11 @@
[ERROR]           >
[ERROR]             <time
[ERROR]               dateTime={821849040000}
[ERROR]               title={821849040000}
[ERROR]             >
[ERROR]     -         22 years ago
[ERROR]     +         23 years ago
[ERROR]             </time>
[ERROR]           </span>
[ERROR]         </div>
[ERROR]         <div>
[ERROR]           <div
[ERROR] 
[ERROR]       46 |       error={{}}
[ERROR]       47 |       errorMessage={''} />);
[ERROR]     > 48 |     expect(wrapper.toJSON()).toMatchSnapshot();
[ERROR]       49 |   });
[ERROR]       50 | });
[ERROR]       51 | 
[ERROR]       
[ERROR]       at Object.<anonymous> (src/components/widgets/WidgetFooter.test.jsx:48:30)
```

Failing Travis CI build: https://travis-ci.org/Graylog2/graylog2-server/builds/404796199#L4323-L4434

--- a/graylog2-web-interface/src/components/widgets/WidgetFooter.test.jsx
+++ b/graylog2-web-interface/src/components/widgets/WidgetFooter.test.jsx
@@ -5,6 +5,15 @@ import WidgetFooter from 'components/widgets/WidgetFooter';
 
 describe('<WidgetFooter />', () => {
   const date = new Date(Date.UTC(1995, 12, 17, 3, 24, 0));
+  let dateNowMock;
+
+  beforeAll(() => {
+    dateNowMock = jest.spyOn(Date, 'now').mockImplementation(() => new Date(Date.UTC(2018, 1, 22, 17, 35, 0)));
+  });
+
+  afterAll(() => {
+    dateNowMock.mockReset();
+  });
 
   it('should render a widget footer locked and enabled replay', () => {
     const wrapper = renderer.create(<WidgetFooter

